Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/EconModulationMultiZone.mo

```diff
@@ -0,0 +1,194 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.AtomicSequences;
+block EconModulationMultiZone
+  "Outdoor and return air damper position modulation sequence for multiple zone VAV AHU"
+
+  parameter Real outDamConSigMax(min=0, max=1, unit="1") = retDamConSigMin
+  "Maximum control loop signal for the outdoor air damper";
+  parameter Real retDamConSigMin(min=0, max=1, unit="1") = 0.5
+  "Minimum control loop signal for the return air damper";
+  parameter Real yConSigMin=0 "Lower limit of controller output";
+  parameter Real yConSigMax=1 "Upper limit of controller output";
+  parameter Real controllerGain=1 "Gain of controller";
+
+  CDL.Interfaces.RealInput TSup(unit="K", quantity = "ThermodynamicTemperature")
+    "Measured supply air temperature" annotation (Placement(transformation(extent={{-160,-40},{-120,0}}),
+        iconTransformation(extent={{-120,50},{-100,70}})));
+  CDL.Interfaces.RealInput TCooSet(unit="K", quantity = "ThermodynamicTemperature")
+    "Supply air temperature cooling setpoint" annotation (Placement(transformation(extent={{-160,-10},{-120,30}}),
+        iconTransformation(extent={{-120,80},{-100,100}})));
+  CDL.Interfaces.RealInput uOutDamPosMin(min=0, max=1, unit="1")
+    "Minimum economizer damper position limit as returned by the EconDamperPositionLimitsMultiZone sequence"
+    annotation (Placement(transformation(extent={{-160,-120},{-120,-80}}),
+        iconTransformation(extent={{-120,-30},{-100,-10}})));    //fixme: add quantity for ALL damper positions?
+  CDL.Interfaces.RealInput uOutDamPosMax(min=0, max=1, unit="1")
+    "Maximum economizer damper position limit as returned by the EconEnableDisableMultiZone sequence. 
+    If the economizer is disabled, this value equals uOutDamPosMin"
+    annotation (Placement(transformation(extent={{-160,-90},{-120,-50}}),
+        iconTransformation(extent={{-120,0},{-100,20}})));
+  CDL.Interfaces.RealInput uRetDamPosMin(min=0, max=1, unit="1")
+    "Minimum return air damper position limit as returned by the EconEnableDisableMultiZone sequence"
+    annotation (Placement(transformation(extent={{-160,40},{-120,80}}),
+        iconTransformation(extent={{-120,-100},{-100,-80}})));
+  CDL.Interfaces.RealInput uRetDamPosMax(min=0, max=1, unit="1")
+    "Maximum return air damper position limit as returned by the EconEnableDisableMultiZone sequence."
```

Comment:
Don't use a period in the comment string. (Also below are a few comments with periods.)